velox/docs/functions/presto/binary.rst

```diff
@@ -101,3 +101,19 @@ Binary Functions
 .. function:: xxhash64(binary) -> varbinary
 
     Computes the xxhash64 hash of ``binary``.
+    
+.. function:: lpad(binary, size, padbinary) -> varbinary
+    :noindex:
```

Comment:
Why is "noindex" needed here ?

Reply:
ive left :noindex: here to match with the presto code. the Varbinary functions have the :noindex: tag however the Varchars don't.